backend/src/nodes/nodes/image/video_frame_iterator.py

```diff
@@ -136,6 +145,7 @@ def run(
                         r=fps,
                         crf=crf,
                         preset=video_preset if video_preset != "none" else None,
+                        vcodec=codec_map[video_type],
```

Comment:
There is no "none" video preset, but there is a "none" video type. Did you add this check for the wrong variable?

Reply:
> Using none won't even get this far in the code

Oops, I didn't see the if at the start of the function. Yes, that's fine.

> And no, this is supposed to be video type

Video preset can't have the value `"none"` though. The dropdown just doesn't have this as an option.

Reply:
There's no way to not specify a name, directory, or quality either. It just gets ignored when you set it to none. 

I wanna get rid of none anyway. The only reason it exists is so you can export to an image sequence using the save image node, but since ffmpeg can export image sequences I figure we can just make that an option instead and make it a bit easier and intuitive

Reply:
Not what I meant. I mean `video_preset if video_preset != "none" else None` is useless because `video_preset` can't be `"none"`.

Reply:
Oooooh, I didn't notice that because it wasn't in the little diff in the comment. Yeah I think I just forgot to remove that when I removed none as an option. But also that's not really related to this PR

Reply:
> that's not really related to this PR

True.

Reply:
Actually, it _is_ in the diff in the comment.... I just didn't notice it lol. I'll fix that when I add the PNG sequence stuff